department-of-veterans-affairs/caseflow
# Remove hardcoded data from Hearing model

Resolves #9595

### Description
Removed hard coded data from Hearing model as specified in issue ticket.

### Acceptance Criteria

### Testing Plan
1. Go to Daily Docket
2. Confirm Hearings for RO display correctly.
